internal/db/schema/migrations/postgres/11/01_server_type_enum.up.sql

```diff
@@ -0,0 +1,24 @@
+begin;
+
+create table server_type_enm (
+  name text primary key
+    constraint only_predefined_server_types_allowed
+      check (
+        name in (
+          'controller',
+          'worker'
+        )
+      )
+);
```

Comment:
Can you add a [comment](https://github.com/hashicorp/boundary/blob/c9e4c877b89038c9e9d4f1edecb15a1123073519/internal/db/schema/migrations/postgres/10/04_vault_credential.up.sql#L135-L137) to the table?

(I left a similar comment in my original review but it wasn't saved)

Reply:
Done!